### PR TITLE
Loop

### PR DESCRIPTION
**Pull request by Gurgenforgit**
loops

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea
 README.md
 out
+java-.iml
```
Comment (TPoghosyan) [inline .gitignore]: you can ignore all .iml files instead of only java-

File: CheckIfInputNumberIsCubeOfANumber.java
Comment (TPoghosyan) [inline CheckIfInputNumberIsCubeOfANumber.java:1]: please in future add class files for each homework in separate package

File: PrintDigitsOfNumberFromLeftToRight.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+public class PrintDigitsOfNumberFromLeftToRight {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println( "Input any positive whole number");
+        int a = sc.nextInt();
+
+        // Preventing input not positive number
+        if (a <= 0){
+            System.out.println(" Next time please input a Positive whole number");
+            System.exit(1);
```
Comment (TPoghosyan) [inline PrintDigitsOfNumberFromLeftToRight.java]: try to not exit application and ask for positive number again if number is negative

Comment (Gurgenforgit) [reply PrintDigitsOfNumberFromLeftToRight.java]: done

File: .gitignore (hunk 1 above)
Comment (Gurgenforgit) [reply .gitignore]: done

File: CheckIfInputNumberIsCubeOfANumber.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input any whole number");
+        int input = sc.nextInt();
+        if (input >= 0) {
+            int i = 0;
+            for (i = 0; i * i * i <= input; i++) {
+                if (i * i * i == input) {
+                    System.out.println(input + " is the cube of " + i);
+                    break;
```
Comment (David91) [inline CheckIfInputNumberIsCubeOfANumber.java:13]: TBD 2

File: CreateArrayFromAnotherArray.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        int a = sc.nextInt();
+        int b = sc.nextInt();
+        if(a < 0 || b < 0 || a == b) {
+            System.out.println("Not valid input for this case");
+            System.exit(1);
+        }
+            // define the bigger
+            int c;
+            int d;
+            if(a < b)
```
Comment (David91) [inline CreateArrayFromAnotherArray.java]: naming convention

File: VariousArrayWithNegativeAndPositiveNumbers.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        double b = arr[0]; // by my logic 'b' should be 0, but if it is 0, the code doesn't run like it should, it skips add the first element
+        int i = 0;
+        for (; i < arr.length; ) {
+            if (arr[i] > 0) {
+                i++;
+                b = b + arr[i];
+            }
+            if (arr[i] < 0) {
+                i = i + 2;
+                b = b + arr[i];
```
Comment (David91) [inline VariousArrayWithNegativeAndPositiveNumbers.java]: exchange iteration step with sum step in both `if` statements.

**Review by David91 — CHANGES_REQUESTED** [review]

**Pull request merged**